source/apiVolontaria/volunteer/models.py

```diff
@@ -1,3 +1,24 @@
 from django.db import models
 
-# Create your models here.
+
+class Cycle(models.Model):
+
+    class Meta:
+        verbose_name_plural = 'Cycles'
+
+    name = models.CharField(
+        verbose_name="Name",
+        max_length=100,
+    )
+
+    start_date = models.DateTimeField(
+        verbose_name="Start date",
+        blank=True,
+        null=True
```

Comment:
Do we want to permit blank/NULL start_date and end_date? Shouldn't those field be filled at creation time?

Reply:
Good question. I thought it can be great to let people set the quantity of data they want and let all fields optionnal. Maybe user just want to create some cycle but do not have exact date for the moment.

It's the same things in Github when you create a Milestone, you can let the `due date` field optional.

Reply:
I'm okay with that. We'll need to take into account that those field can be empty while developping